### PR TITLE
Image file formats

### PR DESCRIPTION
**Pull request by tischi**

### CODE REVIEW

**Comment by tischi** [comment]
@manerotoni 

could you please have a look whether we can merge this?

File: _includes/image_file_formats/activities/image_file_formats_imagejgui.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+### Image inspection
+
+- See activity
+
```
Comment (manerotoni) [inline _includes/image_file_formats/activities/image_file_formats_imagejgui.md:4]: Could you be more specific. Open with drag and drop? using Bioformats and enable show metadata? Using Bigdataviewer (not sure where the metadata is displayed there).
Should one have a look at the original text, xml file?

File: _modules/image_file_formats.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
```
Comment (manerotoni) [inline _modules/image_file_formats.md:1]: See next push for typo changes etc.

File: _includes/image_file_formats/exercises/image_file_formats_imagejgui.md [hunk 3]
```diff
@@ -13,6 +13,13 @@
 
 ### Image saving
```
Comment (manerotoni) [inline _includes/image_file_formats/exercises/image_file_formats_imagejgui.md:14]: Ideally would be nice to show how to save a movie. this is something you embed in a presentation. However, AVI in QuickTime is not read properly (not supported), but is read properly in powerpoint. 
 The movie saving capability of IJ is not that great.

**Review by manerotoni — COMMENTED** [review]
Hi @tischi,
I am done with the review/editing. 
May be add some more specifications in the activity on opening the images (see also my other comment).

File: _includes/image_file_formats/exercises/image_file_formats_imagejgui.md (hunk 3 above)
Comment (tischi) [reply _includes/image_file_formats/exercises/image_file_formats_imagejgui.md:14]: @manerotoni 


There is something already:

`Saving 8 bit single channel movie as GIF: [xyt_8bit__mitocheck_incenp.tif](https://github.com    /NEUBIAS/training-resources/raw/master/image_data/xyt_8bit__mitocheck_incenp.tif)`

I think that's fine for now?!

File: _includes/image_file_formats/activities/image_file_formats_imagejgui.md (hunk 1 above)
Comment (tischi) [reply _includes/image_file_formats/activities/image_file_formats_imagejgui.md:6]: Fine like this?
https://github.com/NEUBIAS/training-resources/pull/198/commits/c8d14a9434269dbddc2f1b52bbdf584a974e3c65

File: _includes/image_file_formats/activities/image_file_formats_imagejgui.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    - Open with a web browser
+      - It should look the same as when you saved it
+
+- Saving 16 bit two channel movie as JPEG: [xyzct_16bit__mitosis.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xyzct_16bit__mitosis.tif)
+  - Select a timepoint in the middle of the movie 
+  - [File > Save As > JPEG...] 
+    - Open JPEG with Fiji
+    - Image dimensions, data type, pixel values, and metadata have changed  
+  
+- Saving 8 bit single channel movie as GIF: [xyt_8bit__mitocheck_incenp.tif](https://github.com/NEUBIAS/training-resources/raw/master/image_data/xyt_8bit__mitocheck_incenp.tif)
```
Comment (tischi) [inline _includes/image_file_formats/activities/image_file_formats_imagejgui.md:34]: @manerotoni This in fact creates a movie, which can be viewed in a web browser.
Should we also add saving as `AVI`?

**Comment by tischi** [comment]
@manerotoni I am merging now, because I need to teach this tomorrow.

**Comment by manerotoni** [comment]
Looks fine. I just made a minor formatting improvement in the activity directly on master.

**Pull request merged**